obal/data/modules/copr_project.py

```diff
@@ -0,0 +1,79 @@
+"""
+Create a Project in Copr
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.copr import copr_cli, CoprCliCommandError, full_name # pylint:disable=import-error,no-name-in-module
+
+
+def project_exists(user, project, module):
+    """
+    Return true if a project already exists for a user
+    """
+    command = [
+        'list',
+        user
+    ]
+
+    try:
+        project_list = copr_cli(command)
+    except CoprCliCommandError as error:
+        module.fail_json(msg='Copr project listing failed', command=' '.join(error.command), output=error.message)
+
+    return "Name: {}".format(project) in project_list
+
+def main():
+    """
+    Create a Project in Copr
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            user=dict(type='str', required=True),
+            project=dict(type='str', required=True),
+            chroots=dict(type='list', required=True),
+            description=dict(type='str', required=False),
+            unlisted_on_homepage=dict(type='bool', required=False, default=True),
+            delete_after_days=dict(type='str', required=False, default=None),
```

Comment:
```suggestion
            delete_after_days=dict(type='str', required=False),
```

There is no need to set `default=None` if there is no default :)